wewatch-client/src/App.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
-import { BrowserRouter as Router, Route } from "react-router-dom";
 
-import Chat from 'components/Chat/Chat';
-import Join from 'components/Join/Join';
+import Chat from './components/Chat/Chat';
```

Comment:
does this work without the './' ? We should have a file called 
jsconfig.json that that has a setting for importing with src being the highest level. so you wouldnt have to make these changes

Reply:
Let me give that a try